Sail/src/Sail/utils/Storage/SettingStorage.cpp

```diff
@@ -197,7 +197,7 @@ void SettingStorage::createGameDefaultMap() {
 	gameSettingsDynamic["map"]["tileSize"] =	DynamicSetting(7.0f, 1.0f, 30.0f);
 	gameSettingsDynamic["map"]["clutter"] = DynamicSetting(0.85f,	0.0f,	5.0f);
 	gameSettingsDynamic["map"]["seed"] =    DynamicSetting(0.0f,	0.0f,	1000000.0f);
-	gameSettingsDynamic["map"]["sprinklerTime"] = DynamicSetting(60.0f, 0.0f, 600.0f);
+	gameSettingsDynamic["map"]["sprinklerTime"] = DynamicSetting(6.0f, 0.0f, 6.1f);
```

Comment:
Lower number of map look-ups:

auto& gameSetting = gameSettingsDynamic["map"]
map["tilesize"] = ...
map["clutter"] = ...
etc